packages/flutter/lib/src/rendering/child_view.dart

```diff
@@ -20,7 +20,7 @@ import 'object.dart';
 mojom.ViewProxy _initViewProxy() {
   int viewHandle = ui.takeViewHandle();
   assert(() {
-    if (viewHandle == 0)
+    if (viewHandle == core.MojoHandle.INVALID)
       debugPrint('Child view are supported only when running in Mojo shell.');
```

Comment:
Why print anything to the console then?

I think Flutter apps should, in correct operation, be completely silent on the console. Thus if we're printing something to the console, that means it's not correct operation. If it's not correct operation, it seems like we should signal a failure, which usually we do by an exception.